ios/BetterSocial/Info.plist

```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>BetterSocial</string>
+	<string>Helio</string>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from "BetterSocial" to "Helio" seems fine as it's a simple string replacement for the app name. However, please ensure that all references to the old app name in the codebase have been updated to avoid any inconsistencies.
```diff
- 	<string>BetterSocial</string>
+ 	<string>Helio</string>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->